src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.cs

```diff
@@ -267,7 +270,7 @@ private static bool IsRenameValid(MutableConflictResolution conflictResolution,
                 }
 
                 // Some types of symbols (namespaces, cref stuff, etc) might not have ContainingAssemblies
-                if (renamedSymbol.ContainingAssembly != null)
+                if (renamedSymbol.ContainingAssembly != null && !renamedSymbol.IsTupleField())
```

Comment:
Would be nice to have a similar comment as earlier for why we are excluding Tuple fields.